engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js

```diff
@@ -28,15 +29,27 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsController
             $scope.panel = {loading: true};
         }
 
-        $scope.contentHost = ContentHost.get({id: $scope.$stateParams.contentHostId}, function (contentHost) {
-            $scope.$watch("contentHostTable.rows.length > 0", function () {
-                $scope.contentHostTable.replaceRow(contentHost);
-            });
+        $scope.contentHost = {$promise: contentHostDeferred.promise};
+        $scope.host = Host.get({id: $scope.$stateParams.contentHostId}, function(host) {
```

Comment:
I know that you've updated the links everywhere to use `host.id` for `$scope.$stateParams.contentHostId` but I think it would be much clearer if we changed the state param name to `$scope.$stateParams.hostId`.